templates.dist/estate/similar_estates.php

```diff
@@ -1,30 +1,46 @@
-<h2><?php esc_html_e('More Estates like this', 'onoffice');?></h2>
-<ul>
-
-<?php
-while ( $currentEstate = $pEstates->estateIterator() ) : ?>
-	<li>
-		<a href="<?php echo esc_url($pEstates->getEstateLink()); ?>">
-			<?php
-			$estatePictures = $pEstates->getEstatePictures();
-			foreach ($estatePictures as $id) : ?>
-				<img src="<?php echo esc_url($pEstates->getEstatePictureUrl
-					($id, ['width' => 200, 'height' => 100])); ?>">
-			<?php endforeach; ?>
-		</a>
-		<p>
-			<?php
-			$fieldsForOutput = ['objektnr_extern', 'wohnflaeche', 'ort'];
-			echo $currentEstate['objekttitel'].'<br>';
-			foreach ($fieldsForOutput as $field) {
-				$value = $currentEstate[$field];
-				echo esc_html($pEstates->getFieldLabel($field).': '
-					.(is_array($value) ? esc_html(implode(', ', $value)) : esc_html($value))).'<br>';
-			}
-			?>
-		</p>
-	</li>
-<?php endwhile; ?>
-
-</ul>
-<br>
\ No newline at end of file
+<div class="oo-detailsheadline">
+	<h2><?php esc_html_e('More Estates like this', 'onoffice');?></h2>
+</div>
+<div class="oo-listframe" id="oo-similarframe">
+	<?php
+	while ( $currentEstate = $pEstates->estateIterator() ) { ?>
+		<div class="oo-listobject">
+			<div class="oo-listobjectwrap">
+				<?php
+				$estatePictures = $pEstates->getEstatePictures();
+				foreach ( $estatePictures as $id ) {
+					$pictureValues = $pEstates->getEstatePictureValues( $id );
+					echo '<a href="'.$pEstates->getEstateLink().'" style="background-image: url('.$pEstates->getEstatePictureUrl( $id ).');" class="oo-listimage">';
+					if ($pictureValues['type'] === \onOffice\WPlugin\Types\ImageTypes::TITLE && $marketingStatus != '') {
+						echo '<span>'.esc_html($marketingStatus).'</span>';
```

Comment:
Where does `$marketingStatus` come from over here? If I'm seeing it right it is the marketing status of the estate whose detail view the similar estates view is embedded in.
`$marketingStatus` needs to be redefined inside this inner loop, right?

Reply:
Yes, you are right. I changed it for my next commit.

Reply:
Don't hesitate to commit changes more often in smaller commits!

Reply:
Commited!